inlong-audit/audit-service/src/main/java/org/apache/inlong/audit/selector/impl/DBDataSource.java

```diff
@@ -29,28 +32,17 @@
 import java.text.MessageFormat;
 import java.util.concurrent.atomic.AtomicInteger;
 
-import static config.ConfigConstants.CACHE_PREP_STMTS;
-import static config.ConfigConstants.MAX_INIT_COUNT;
-import static config.ConfigConstants.PREP_STMT_CACHE_SIZE;
-import static config.ConfigConstants.PREP_STMT_CACHE_SQL_LIMIT;
-import static config.SqlConstants.IS_LEADER_SQL;
-import static config.SqlConstants.RELEASE_SQL;
-import static config.SqlConstants.REPLACE_LEADER_SQL;
-import static config.SqlConstants.SEARCH_CURRENT_LEADER_SQL;
-import static config.SqlConstants.SELECTOR_SQL;
-import static config.SqlConstants.SELECT_TEST_SQL;
-
 /**
  * DB data source
  */
 public class DBDataSource {
 
     private static final Logger logger = LoggerFactory.getLogger(DBDataSource.class);
-    private String selectorSql = SELECTOR_SQL;
-    private String replaceLeaderSql = REPLACE_LEADER_SQL;
-    private String reLeaseSql = RELEASE_SQL;
-    private String isLeaderSql = IS_LEADER_SQL;
-    private String searchCurrentLeaderSql = SEARCH_CURRENT_LEADER_SQL;
+    private String selectorSql = SqlConstants.SELECTOR_SQL;
+    private String replaceLeaderSql = SqlConstants.REPLACE_LEADER_SQL;
+    private String reLeaseSql = SqlConstants.RELEASE_SQL;
+    private String isLeaderSql = SqlConstants.IS_LEADER_SQL;
+    private String searchCurrentLeaderSql = SqlConstants.SEARCH_CURRENT_LEADER_SQL;
```

Comment:
Using the constants from `SqlConstants` is enough, no need to create new variables

Reply:
These member variables will be updated, so there are no direct references to constants.

Reply:
Got it.